microsoft-onedrive/1.0/modules/ROOT/pages/microsoft-onedrive-connector-reference.adoc

```diff
@@ -0,0 +1,1317 @@
+
+Gives read or write access to a specified file for users that are specified in the *Permissions Info* field.
+
+=== Parameters
+
+[%header%autowidth.spread]
+|===
+| Name | Type | Description | Default Value | Required
+| Configuration | String | Name of the configuration to use. | | x
+| DriveItem ID a| String |  ID of the drive DriveItem to which to add this permission.  |  | x
+| Permissions Info a| Any |  Information about the permission to add. |  #[payload] | 
+| Read Timeout a| Number |  Read timeout value. Used to override the read timeout values defined in the connector configuration. |  | 
+| Read Timeout Time Unit a| Enumeration, one of:
+
+** NANOSECONDS
+** MICROSECONDS
+** MILLISECONDS
+** SECONDS
+** MINUTES
+** HOURS
+** DAYS |  Time unit value for the *Read Timeout* field. |  | 
+| Config Ref a| ConfigurationProvider | Name of the configuration to use to execute this component. |  | x
+| Streaming Strategy a| * <<RepeatableInMemoryStream>>
+* <<RepeatableFileStoreStream>>
+* non-repeatable-stream |  Configures how Mule processes streams. The default is to use repeatable streams. |  | 
+| Location a| Enumeration, one of:
+
+** GROUP
+** USER |  Location from which to retrieve the list of drive IDs. |  | 
+| Entity ID a| String |  ID of the entity to use to retrieve a list of drives. |  | 
+| Drive ID a| String |  Drive ID used to locate a DriveItem. |  | 
+| Target Variable a| String |  Name of the variable that stores the operation's output. |  | 
+| Target Value a| String | Expression that evaluates the operation’s output. The outcome of the expression is stored in the *Target Variable* field. |  #[payload] | 
+| Reconnection Strategy a| * <<Reconnect>>
+* <<ReconnectForever>> |  A retry strategy in case of connectivity errors. |  | 
+|===
+
+=== Output
+
+[%autowidth.spread]
+|===
+|Type |Binary
+| Attributes Type a| Binary
+|===
+
+=== For Configurations
+
+* <<OnedriveConfig>> 
+
+=== Throws
+
+* MICROSOFTONEDRIVE:BAD_REQUEST 
+* MICROSOFTONEDRIVE:CONFLICT 
+* MICROSOFTONEDRIVE:CONNECTIVITY 
+* MICROSOFTONEDRIVE:FORBIDDEN 
+* MICROSOFTONEDRIVE:INSUFFICIENT_STORAGE 
+* MICROSOFTONEDRIVE:INVALID_CONNECTION 
+* MICROSOFTONEDRIVE:INVALID_INPUT 
+* MICROSOFTONEDRIVE:LOCKED 
+* MICROSOFTONEDRIVE:NOT_FOUND 
+* MICROSOFTONEDRIVE:RETRY_EXHAUSTED 
+* MICROSOFTONEDRIVE:SERVER_ERROR 
+* MICROSOFTONEDRIVE:TIMEOUT 
+* MICROSOFTONEDRIVE:TOO_MANY_REQUESTS 
+* MICROSOFTONEDRIVE:UNAUTHORIZED 
+
+
+[[CopyDriveItem]]
+== Copy Drive Item
+`<microsoftOneDrive:copy-drive-item>`
+
+Executes a copy request to Microsoft OneDrive.
+
+=== Parameters
+
+[%header%autowidth.spread]
+|===
+| Name | Type | Description | Default Value | Required
+| Configuration | String | Name of the configuration to use. | | x
+| DriveItem ID a| String | ID of the driveItem to copy. |  | x
+| Parent DriveItem ID a| String | ID of the folder to copy the selected driveItem to. If unspecified, the selected file is copied in the same folder in which it resides. |  | 
```

Comment:
Maybe:

...is duplicated in the same folder in which it resides.

...is copied to the same folder in which it resides.